File: ProblemSolving/C/1005/Program.c
#include <stdio.h>

int main() {
    double a, b;
    scanf("%lf",&a);
    scanf("%lf",&b);

    double media = (a * 3.5 + b * 7.5) / 11;

    printf("MEDIA = %.5lf\n", media);

    return 0; 
}

File: ProblemSolving/CSharp/1005/Program.cs
using System; 
using System.Globalization;

class _1005 
{
    static void Main(string[] args) 
    { 
        double a = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        double b = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);

        var media = (a*3.5 + b*7.5) / 11;

        Console.WriteLine($"MEDIA = {media.ToString("f5")}", CultureInfo.InvariantCulture);
    }
}